freqtrade/strategy/interface.py

```diff
@@ -338,6 +338,32 @@ def informative_pairs(self) -> ListPairsWithTimeframes:
 # END - Intended to be overridden by strategy
 ###
 
+    def gather_informative_pairs(self) -> ListPairsWithTimeframes:
+        """
+        Internal method which gathers all informative pairs (user or automatically defined).
+        """
+        informative_pairs = self.informative_pairs()
+        for attr_name in dir(self.__class__):
+            cls_attr = getattr(self.__class__, attr_name)
+            if not callable(cls_attr):
+                continue
+            if not getattr(cls_attr, '_is_informative', False):
+                continue
+            timeframe = getattr(cls_attr, '_timeframe', None)
+            asset = getattr(cls_attr, '_asset', None)
+            assert asset is not None
+            assert timeframe is not None
```

Comment:
Please don't use assert in non-test code.
Few know it - but python has a "production-mode" flag which makes python skip all assert statements.

It'll also cause a non-cought exception.